setup.sh

```diff
@@ -1,29 +1,69 @@
 #!/bin/bash
 cp ./.env.example ./.env
+echo -e "\nNEXT_PUBLIC_API_BASE_URL=$1"  >> ./.env
```

Comment:
Can be moved to the web and space directories let's completely remove the root `.env.example`